python/doc/theory/probabilistic_modeling/trend_transform.rst

```diff
@@ -0,0 +1,62 @@
+Trend computation
+-----------------
+
+A multivariate stochastic process
```

Comment:
Here, the resulting process when the trend has been removed is that it is centered and not stationary. Please modify X_{stat} in X_{c} in the text and remove the stationary hypothesis.
Furthermore, the action of adding a trend to a process can be applied to any process!!!

Reply:
The TrendTransform class enables to add and remove a deterministic function (called trend) to a process. 
Please modify the text to make it more general : we create the process Y(\omega, t) = X(\omega, t) + f(t)
When f is estimated estimated from a realization of the process Y, the resulting process X is centered: E[X(\omega, t)} = 0 for all t